src/index.html

```diff
@@ -5,10 +5,78 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="nav__logo" href="#home">
+        <img
+        src="/images/logo.png" 
+        alt="logo" 
+        class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+            href="#apple" 
+            class="nav__link is-active"
+            >Apple</a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#sumsung" 
+            class="nav__link"
+            >Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#phones" 
+            class="nav__link"
+            >Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#laptop" 
+            class="nav__link" 
+            data-qa="hover"
+            >Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#technic" 
+            class="nav__link"
+            >Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#tablet" 
+            class="nav__link"
+            >Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#photo" 
+            class="nav__link"
+            >Фото</a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#video" 
+            class="nav__link"
+            >Видео</a>
+          </li>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), the 3rd item